docs-v2/integrations/all/canny.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Canny
+sidebarTitle: Canny
+---
+
+API configuration: [`canny`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (API Key)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [API Reference](https://developers.canny.io/api-reference)
+-   [Authentication](https://developers.canny.io/api-reference#authentication)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+- Canny's API docs only mention POST requests, but the API's endpoints also accept GET requests.
+- For GET requests, the API accepts any piece of body data as a URL param.
+
+<Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/calendly.mdx).</Note>
```

Comment:
```suggestion
<Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/canny.mdx).</Note>
```

Reply:
@keedyc, the frontend needs to be rebuilt. Perhaps @bodinsamuel can chime in here.